Index: app/src/main/java/fr/nebulo9/brokemap/ui/composables/sections/PlaceTypes.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package fr.nebulo9.brokemap.ui.composables.sections\r\n\r\nimport android.graphics.drawable.Icon\r\nimport android.media.Image\r\nimport androidx.compose.material.icons.Icons\r\nimport androidx.compose.material.icons.filled.*\r\nimport androidx.compose.ui.graphics.vector.ImageVector\r\nimport fr.nebulo9.brokemap.R\r\n\r\ndata class PlaceType(\r\n    val id: String,\r\n    val label: String,\r\n    val icon: ImageVector\r\n)\r\n\r\nval placeTypes = listOf(\r\n    PlaceType(\"restaurant\", \"Restaurant\", Icons.Default.Place),\r\n    PlaceType(\"bar\", \"Bar\", Icons.Default.Place),\r\n    PlaceType(\"dancingBar\", \"Dancing Bar\", Icons.Default.Place),\r\n    PlaceType(\"Museum\", \"Museum\", Icons.Default.Place),\r\n    PlaceType(\"bench\", \"Bench\", Icons.Default.Place),\r\n    PlaceType(\"fastFood\", \"Fast Food\", Icons.Default.Place),\r\n)
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/fr/nebulo9/brokemap/ui/composables/sections/PlaceTypes.kt b/app/src/main/java/fr/nebulo9/brokemap/ui/composables/sections/PlaceTypes.kt
--- a/app/src/main/java/fr/nebulo9/brokemap/ui/composables/sections/PlaceTypes.kt	(revision 01e87e37c4af76d8be4337fa37794d20ffa7e1c1)
+++ b/app/src/main/java/fr/nebulo9/brokemap/ui/composables/sections/PlaceTypes.kt	(date 1772916454864)
@@ -14,10 +14,11 @@
 )
 
 val placeTypes = listOf(
-    PlaceType("restaurant", "Restaurant", Icons.Default.Place),
-    PlaceType("bar", "Bar", Icons.Default.Place),
-    PlaceType("dancingBar", "Dancing Bar", Icons.Default.Place),
-    PlaceType("Museum", "Museum", Icons.Default.Place),
-    PlaceType("bench", "Bench", Icons.Default.Place),
-    PlaceType("fastFood", "Fast Food", Icons.Default.Place),
+    PlaceType("restaurant", "Restaurant", Icons.Default.Restaurant),
+    PlaceType("bar", "Bar", Icons.Default.LocalBar),
+    PlaceType("dancing_bar", "Dancing Bar", Icons.Default.Nightlife),
+    PlaceType("museum", "Museum", Icons.Default.Museum),
+    PlaceType("bench", "Bench", Icons.Default.Chair),
+    PlaceType("fastfood", "Fast Food", Icons.Default.Fastfood),
+
 )
\ No newline at end of file
