Index: app/src/main/java/fr/nebulo9/brokemap/MainActivity.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package fr.nebulo9.brokemap\r\n\r\nimport android.os.Bundle\r\nimport androidx.activity.ComponentActivity\r\nimport androidx.activity.compose.setContent\r\nimport androidx.activity.enableEdgeToEdge\r\nimport fr.nebulo9.brokemap.ui.composables.screens.MapScreen\r\nimport fr.nebulo9.brokemap.ui.theme.BrokeMapTheme\r\n\r\nclass MainActivity : ComponentActivity() {\r\n    override fun onCreate(savedInstanceState: Bundle?) {\r\n        super.onCreate(savedInstanceState)\r\n        enableEdgeToEdge()\r\n        setContent {\r\n            BrokeMapTheme {\r\n                MapScreen()\r\n            }\r\n        }\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/fr/nebulo9/brokemap/MainActivity.kt b/app/src/main/java/fr/nebulo9/brokemap/MainActivity.kt
--- a/app/src/main/java/fr/nebulo9/brokemap/MainActivity.kt	(revision 3b296e42b596240585aae002d7a9e71ed6d69d23)
+++ b/app/src/main/java/fr/nebulo9/brokemap/MainActivity.kt	(date 1772440660524)
@@ -1,19 +1,43 @@
 package fr.nebulo9.brokemap
 
+import android.location.Location
 import android.os.Bundle
 import androidx.activity.ComponentActivity
 import androidx.activity.compose.setContent
 import androidx.activity.enableEdgeToEdge
-import fr.nebulo9.brokemap.ui.composables.screens.MapScreen
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import com.google.android.gms.location.FusedLocationProviderClient
+import com.google.android.gms.location.LocationServices
+import com.google.android.gms.maps.model.CameraPosition
+import com.google.android.gms.maps.model.LatLng
+import com.google.maps.android.compose.CameraPositionState
+import com.google.maps.android.compose.GoogleMap
+import com.google.maps.android.compose.rememberCameraPositionState
 import fr.nebulo9.brokemap.ui.theme.BrokeMapTheme
 
 class MainActivity : ComponentActivity() {
+    private lateinit var fusedLocationClient: FusedLocationProviderClient
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        var defaultLocation = LatLng(.0,.0)
         enableEdgeToEdge()
         setContent {
             BrokeMapTheme {
-                MapScreen()
+                Scaffold(modifier = Modifier.fillMaxSize()) { innerPadding ->
+                    val cameraPositionState = rememberCameraPositionState {
+                        position = CameraPosition.fromLatLngZoom(defaultLocation, 10f)
+                    }
+                    GoogleMap(
+                        modifier = Modifier.fillMaxSize(),
+                        cameraPositionState = cameraPositionState
+                    )
+                }
             }
         }
     }
